src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -258,13 +258,13 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
 
   def checkUserInRawls(
     pipeline: WithTransformerConcatenation[HttpRequest, Future[HttpResponse]],
-    requestContext: RequestContext): Future[PerRequestMessage] = {
+    requestContext: RequestContext, userId: String): Future[PerRequestMessage] = {
```

Comment:
ahh, i see. so when I call the checkUserInRawls function in line 77, it doesn't yet know what the userId is for this Authorization header because it hasn't passed through the proxy. so i would assume this means it's ok if I pass in userId here instead of deriving it from the pipeline?